SMLHelper/Patchers/TechTypePatcher.cs

```diff
@@ -107,6 +105,8 @@ private static List<int> PreRegisteredTechTypes()
 
         internal static void Patch(HarmonyInstance harmony)
         {
+            cacheManager.SaveCache();
```

Comment:
What happens if saving fails here?

Reply:
And also the same question on the CraftTreeTypePatcher

Reply:
Well, nothing.